config/teleop.yaml

```diff
@@ -4,20 +4,28 @@ teleop:
   ra_controls:
     joint_a:
       multiplier: 1
+      slow_mode_multiplier: 0.5
```

Comment:
Not sure why the slow mode changes made it to this PR but we should probably fix that.